src/services/resolver/strategy/ResolvingStrategy.php

```diff
@@ -11,5 +11,5 @@
 namespace PharIo\Phive;
 
 interface ResolvingStrategy {
-    public function execute(RequestedPharResolverService $pharResolverService): void;
+    public function execute(RequestedPharResolverService $service): void;
```

Comment:
What's the motivation for this?

Reply:
Since the introduction of the named parameters in PHP, Psalm checks that the parameters of a class have the same name than the ones from the interface implemented.

In this case there are several classes that implement this interface, and all of them have this parameter named `$service`.
I thought that changing the interface instead of the classes was preferable in order to reduce the diff/noise.

Reply:
Point taken.